numpy/distutils/command/egg_info.py

```diff
@@ -1,9 +1,17 @@
 from __future__ import division, absolute_import, print_function
 
+import sys
+
 from setuptools.command.egg_info import egg_info as _egg_info
 
 class egg_info(_egg_info):
     def run(self):
+        if 'sdist' in sys.argv:
+            import warnings
+            warnings.warn("`build_src` is being run, this may lead to missing "
```

Comment:
In the ``skimage`` ``setup.py`` you don't have any ``sdist`` handling, so you also have to add something like ``cmdclass={'sdist': sdist}`` or (better) the ``sdist_checked`` version that numpy and scipy have.